instrumentation/spring/spring-boot-autoconfigure/src/main/java/io/opentelemetry/instrumentation/spring/autoconfigure/exporters/internal/ExporterConfigEvaluator.java

```diff
@@ -19,6 +19,10 @@ private ExporterConfigEvaluator() {}
   public static boolean isExporterEnabled(
       Environment environment, String exportersKey, String wantExporter, boolean defaultValue) {
 
+    if (environment.getProperty("otel.sdk.disabled", Boolean.class, false)) {
```

Comment:
I would add a comment, something like this: the method is used to enable some Spring autoconfigurations

Reply:
putting this condition on each exporter's `@Configuration` could make it more visible and consistent (where we just throw the `@SdkEnabled` condition on everything)

Reply:
great idea - done :smile:

Reply:
> putting this condition on each exporter's `@Configuration` could make it more visible and consistent (where we just throw the `@SdkEnabled` condition on everything)

Should it not be another property configuration allowing to disable the export? 

By setting `otel.sdk.disabled=true`, I understand disable the OpenTelemtry SDK, and so disable  the instrumentation features and the export.

Reply:
> > putting this condition on each exporter's `@Configuration` could make it more visible and consistent (where we just throw the `@SdkEnabled` condition on everything)
> 
> Should it not be another property configuration allowing to disable the export?

you still have `otel.traces.exporter=none` (similar for logs and metrics)

> By setting `otel.sdk.disabled=true`, I understand disable the OpenTelemtry SDK, and so disable the instrumentation features and the export.

yes, `otel.sdk.disabled=true` disables everything at once.